formatters/coverage.go

```diff
@@ -47,3 +47,11 @@ func (c *Coverage) UnmarshalJSON(text []byte) error {
 	*c = cc
 	return nil
 }
+
+func (c *Coverage) AppendNulls(quantity int) Coverage {
```

Comment:
Nice, I'd seen this basic pattern a bunch of places, we can probably use it  across a bunch of formatters later!